### ISSUE TITLE
undefined is not an object (evaluating 'u.get.call') since 15.6.0

### ISSUE THREAD

**Issue by krazik** [facebook/react#9961]
Since updating to 15.6.0 I've started having an uncaught exception `undefined is not an object (evaluating 'u.get.call')`  I've pinned version back to `^15.5.0` and problem has gone away

Per Rollbar problem seems me mostly ios safari (9.x?), but sometimes osx safari too.  We've been unable to repo ourselves, but users are clearly experiencing it.

User agents:
```
Mozilla/5.0 (iPhone; CPU iPhone OS 9_2_1 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13D15 Safari/601.1
Mozilla/5.0 (iPad; CPU OS 9_3_5 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13G36 Safari/601.1
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/601.7.7 (KHTML, like Gecko) Version/9.1.2 Safari/601.7.7
```

Stacktraces:
```
TypeError: undefined is not an object (evaluating 'u.get.call')
1 File "webpack:///./~/react-dom/lib/inputValueTracking.js" line 70 col 1 in get
2 File "webpack:///./~/react-dom/lib/inputValueTracking.js" line 36 col 1 in u
3 File "webpack:///./~/react-dom/lib/inputValueTracking.js" line 104 col 1 in updateValueIfChanged
4 File "webpack:///./~/react-dom/lib/ChangeEventPlugin.js" line 99 col 1 in c
5 File "webpack:///./~/react-dom/lib/ChangeEventPlugin.js" line 231 col 1 in b
6 File "webpack:///./~/react-dom/lib/ChangeEventPlugin.js" line 293 col 1 in extractEvents
7 File "webpack:///./~/react-dom/lib/EventPluginHub.js" line 219 col 1 in extractEvents
8 File "webpack:///./~/react-dom/lib/ReactEventEmitterMixin.js" line 26 col 1 in handleTopLevel
9 File "webpack:///./~/react-dom/lib/ReactEventListener.js" line 72 col 1 in i
10 File "webpack:///./~/react-dom/lib/Transaction.js" line 143 col 1 in perform
11 File "webpack:///./~/react-dom/lib/ReactDefaultBatchingStrategy.js" line 62 col 1 in batchedUpdates
12 File "webpack:///./~/react-dom/lib/ReactUpdates.js" line 97 col 1 in i
13 File "webpack:///./~/react-dom/lib/ReactEventListener.js" line 147 col 1 in dispatchEvent
14 File https://d37gvrvc0wt4s1.cloudfront.net/js/v1.9/rollbar.min.js line 1 col 13790 in _wrapped
```
```
TypeError: undefined is not an object (evaluating 'u.get.call')
1 File "webpack:///./~/react-dom/lib/inputValueTracking.js" line 70 col 1 in get
2 File "webpack:///./~/react-dom/lib/ReactDOMInput.js" line 162 col 8 in updateWrapper
3 File "webpack:///./~/react-dom/lib/ReactDOMComponent.js" line 767 col 1 in updateComponent
4 File "webpack:///./~/react-dom/lib/ReactDOMComponent.js" line 722 col 1 in receiveComponent
5 File "webpack:///./~/react-dom/lib/ReactReconciler.js" line 124 col 1 in receiveComponent
6 File "webpack:///./~/react-dom/lib/ReactChildReconciler.js" line 109 col 1 in updateChildren
7 File "webpack:///./~/react-dom/lib/ReactMultiChild.js" line 211 col 1 in _reconcilerUpdateChildren
8 File "webpack:///./~/react-dom/lib/ReactMultiChild.js" line 310 col 1 in _updateChildren
9 File "webpack:///./~/react-dom/lib/ReactMultiChild.js" line 297 col 1 in updateChildren
10 File "webpack:///./~/react-dom/lib/ReactDOMComponent.js" line 942 col 1 in _updateDOMChildren
11 File "webpack:///./~/react-dom/lib/ReactDOMComponent.js" line 760 col 1 in updateComponent
12 File "webpack:///./~/react-dom/lib/ReactDOMComponent.js" line 722 col 1 in receiveComponent
13 File "webpack:///./~/react-dom/lib/ReactReconciler.js" line 124 col 1 in receiveComponent
14 File "webpack:///./~/react-dom/lib/ReactChildReconciler.js" line 109 col 1 in updateChildren
15 File "webpack:///./~/react-dom/lib/ReactMultiChild.js" line 211 col 1 in _reconcilerUpdateChildren
16 File "webpack:///./~/react-dom/lib/ReactMultiChild.js" line 310 col 1 in _updateChildren
17 File "webpack:///./~/react-dom/lib/ReactMultiChild.js" line 297 col 1 in updateChildren
18 File "webpack:///./~/react-dom/lib/ReactDOMComponent.js" line 942 col 1 in _updateDOMChildren
19 File "webpack:///./~/react-dom/lib/ReactDOMComponent.js" line 760 col 1 in updateComponent
20 File "webpack:///./~/react-dom/lib/ReactDOMComponent.js" line 722 col 1 in receiveComponent
21 File "webpack:///./~/react-dom/lib/ReactReconciler.js" line 124 col 1 in receiveComponent
22 File "webpack:///./~/react-dom/lib/ReactCompositeComponent.js" line 753 col 1 in _updateRenderedComponent
23 File "webpack:///./~/react-dom/lib/ReactCompositeComponent.js" line 723 col 1 in _performComponentUpdate
24 File "webpack:///./~/react-dom/lib/ReactCompositeComponent.js" line 644 col 1 in updateComponent
25 File "webpack:///./~/react-dom/lib/ReactCompositeComponent.js" line 560 col 1 in performUpdateIfNecessary
26 File "webpack:///./~/react-dom/lib/ReactReconciler.js" line 156 col 1 in performUpdateIfNecessary
27 File "webpack:///./~/react-dom/lib/ReactUpdates.js" line 150 col 1 in u
28 File "webpack:///./~/react-dom/lib/Transaction.js" line 143 col 1 in perform
29 File "webpack:///./~/react-dom/lib/Transaction.js" line 143 col 1 in perform
30 File "webpack:///./~/react-dom/lib/ReactUpdates.js" line 89 col 1 in perform
31 File "webpack:///./~/react-dom/lib/ReactUpdates.js" line 172 col 1 in T
32 File "webpack:///./~/react-dom/lib/Transaction.js" line 209 col 1 in closeAll
33 File "webpack:///./~/react-dom/lib/Transaction.js" line 156 col 1 in perform
34 File "webpack:///./~/react-dom/lib/ReactDefaultBatchingStrategy.js" line 62 col 1 in batchedUpdates
35 File "webpack:///./~/react-dom/lib/ReactUpdates.js" line 97 col 1 in i
36 File "webpack:///./~/react-dom/lib/ReactEventListener.js" line 147 col 1 in dispatchEvent
37 File https://d37gvrvc0wt4s1.cloudfront.net/js/v1.9/rollbar.min.js line 1 col 13790 in _wrapped
```

**Comment by gaearon** [comment]
This was just reported two times 😉 
The fix will be out soon.

https://github.com/facebook/react/issues/9956

**Comment by gaearon** [comment]
Should be fixed in 15.6.1. Please verify.
